contracts/system_addresses.go

```diff
@@ -19,6 +19,8 @@ var (
 	ChildERC20Contract = types.StringToAddress("0x1003")
 	// ChildERC20PredicateContract is an address of child ERC20 predicate contract on the child chain
 	ChildERC20PredicateContract = types.StringToAddress("0x1004")
+	// EIP1559BurnContract is an address of EIP-1559 burn contract on the child chain
+	EIP1559BurnContract = types.StringToAddress("0x1005")
```

Comment:
WDYT about deploying it to the `0x104` address? Asking this, because I'm going to deploy some other bridge-related smart contracts to this address range ([0x1005...0x1008]): https://github.com/0xPolygon/polygon-edge/pull/1322/files#diff-cb0d9c0c8a217bb409554571a2a979efd58f1dbf13e059824db4b1df98abed3cR24-R30. 

I think it is a good idea to have bridge contracts clustered in the same range. Not sure who is deciding about addresses, but maybe to raise that topic within the team.

Reply:
Done